### DISCUSSION TITLE
Set password for an on-boarded account

### DISCUSSION THREAD

**Discussion by JohnnyLeuthard** [pspete/psPAS#486] [Q&A] [answered]
I'm looking for but can't find if there is a Cmdlet for this but I want to set the password of an on-boarded account in the vault only. 
Here is the real issue
IP change so some vault ID's are pointed to the wrong IUP. I can change the IP, change the value stored in the vault (Like if you went into PrivateArk and changed the vault ID), but I need to have the value stored in the PVWA for retrieval match that value.  I may be taking the long route but what I am trying to do
Getting a list of accounts with the wrong IP; changing the IP; If it is a vault account,  setting a new password in the vault; re-enabling the CPM management; changing the value in the vault only (as you would in PVWA vault only) so it matches what I changed the Vault to; un-suspend (if suspended) the account. Then schedule a password rotation. I haven't found a cmdlet to update the password value of an on-boarded account. The rest i think I have it down or can learn my way through it.

**Comment by pspete** [comment, marked as answer]
You should be able to update password in the vault with this command & parameterset.

https://pspas.pspete.dev/commands/Invoke-PASCPMOperation#passwordupdate

**Reply by JohnnyLeuthard** [reply]
Perfect, that looks like exactly what I needed. Thank you.
